mne/viz/backends/_pyvista.py

```diff
@@ -196,7 +196,10 @@ def __init__(self, fig=None, size=(600, 600), bgcolor='black',
         self.antialias = _get_3d_option('antialias')
         self.depth_peeling = _get_3d_option('depth_peeling')
         # smooth_shading=True fails on MacOS CIs
-        self.smooth_shading = _get_3d_option('smooth_shading')
+        if sys.platform == 'darwin':
+            self.smooth_shading = _get_3d_option('smooth_shading')
+        else:
+            self.smooth_shading = smooth_shading
```

Comment:
It seems like we should have `smooth_shading=None` in the signature, then here:
```
if smooth_shading is None:
    smooth_shading = _get_3d_option('smooth_shading')
```
and then if CIs have a problem with smooth shading we should for them set `MNE_3D_OPTION_SMOOTH_SHADING=false` or whatever.

Reply:
Ok. smooth_shading defaults to True in several class/method signatures, do you want them all to be None, or just here?

Reply:
If it's public then None makes sense. Basically if a user hasn't explicitly set smooth_shading, it should use the _get_3d_option under the hood